lib/curses.neon

```diff
@@ -194,6 +450,7 @@ DECLARE NATIVE FUNCTION standout()
 DECLARE NATIVE FUNCTION wstandout(win: Window)
 DECLARE NATIVE FUNCTION start_color()
 DECLARE NATIVE FUNCTION stdscr(): Window
+DECLARE NATIVE FUNCTION curscr(): Window
```

Comment:
To be consistent with the order of functions already in this file, this should be declared right before `curses_version`.